UnityProject/Assets/Scripts/Cargo/ExportScanner.cs

```diff
@@ -0,0 +1,33 @@
+using UnityEngine;
+
+public class ExportScanner : MonoBehaviour, ICheckedInteractable<HandApply>
+{
+	public bool WillInteract(HandApply interaction, NetworkSide side)
+	{
+		if (!DefaultWillInteract.Default(interaction, side))
+		{
+			return false;
+		}
+
+		if (interaction.HandObject != gameObject)
+		{
+			return false;
+		}
+
+		return true;
+	}
+
+	public void ServerPerformInteraction(HandApply interaction)
+	{
+		var attributes = interaction.TargetObject.GetComponent<ItemAttributesV2>();
+		var price = attributes ? attributes.GetExportCost() : 0;
+		var exportName = interaction.TargetObject.ExpensiveName();
+		var message = price > 0
+			? $"Scanned { exportName }, value: { price } credits."
+			: $"Scanned { exportName }, no export value.";
+
+		// TODO #2400 if it has contents say contents included.
```

Comment:
thanks for creating a TODO and corresponding ticket for it. I updated that ticket with the info on how to get the contents but you can do that in future PR if you don't want to do it now.

Reply:
I'll do it in a future PR, don't want to hold this one up too much so bod can get base attribute stuff in smoother